03/11/2020 07:05:08 - WARNING - __main__ -   Process rank: -1, device: cpu, n_gpu: 0, distributed training: False, 16-bits training: False
Traceback (most recent call last):
  File "/scratch/fs45/nlu/code/transformers/examples/run_glue.py", line 701, in <module>
    main()
  File "/scratch/fs45/nlu/code/transformers/examples/run_glue.py", line 610, in main
    raise ValueError("Task not found: %s" % (args.task_name))
ValueError: Task not found: boolq
